dosomething/fastly-frontend/main.tf

```diff
@@ -37,18 +37,6 @@ resource "fastly_service_v1" "frontend" {
     statement = "req.http.host == \"${var.assets_domain}\""
   }
 
-  condition {
-    type      = "REQUEST"
-    name      = "timed-synthetic-takeover"
-    statement = "req.http.X-Timed-Synthetic-Response == \"true\""
-  }
-
-  response_object {
-    name              = "timed synthetic takeover"
-    request_condition = "timed-synthetic-takeover"
-    content           = file("${path.module}/takeovers/election.html")
-  }
-
```

Comment:
This is from an old homepage takeover. 

It's something that we could re-add (from Git history) if we were to run again.